client/verta/verta/_internal_utils/_artifact_utils.py

```diff
@@ -248,8 +252,8 @@ def serialize_model(model):
         try:  # attempt to deserialize
             reset_stream(model)  # reset cursor to beginning in case user forgot
             model = deserialize_model(model.read())
-        except pickle.UnpicklingError:  # unrecognized model
-            bytestream, _ = ensure_bytestream(model)  # pass along file-like
```

Comment:
This `ensure_bytestream()` was

- problematic, because PySpark models have a `read()`/`write()` interface and bungle up our file-handling logic
- unnecessary, because all appropriate code paths in this method end with an `ensure_bytestream()` further down